test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1183,6 +1200,26 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
 
     #endregion charset encoding tests
 
+    #region Client Certificate Authentication
+
+    It "Verifies Invoke-WebRequest Certificate Authentication Fails without -Certificate"  {
+        $uri = 'https://localhost:8443/'
+        $result = Invoke-WebRequest -Uri $uri -SkipCertificateCheck
+        $failedMessage = 'Status: Failed'
+        $result.RawContent  | Should Match ([regex]::Escape($failedMessage ))
+    }
+
+    It "Verifies Invoke-WebRequest Certificate Authentication Successful with -Certificate" {
```

Comment:
@TravisEz13 Should I add comments in the tests? I have the behavior of the server documented in README.md for the `ClientCertificateCheck` app.